### PR TITLE
Added local storage

### PR DESCRIPTION
**Pull request by gzamudio**
### Summary
- Added local storage
- Added navbar component

### Trello Card
- https://trello.com/c/JTmyqOjt

### CODE REVIEW

File: wolox-books/src/components/Login.vue [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
+  form.column.container(@submit.prevent='submit')
+    img.wolox-img(src='@/assets/LogoWolox.png')
     span.books-title
       | BOOKS
-    label.label
+    label.label(for='email')
       | Email
-    input.input(type='text' v-model='$v.email.$model')
-    .error(v-if="$v.email.$error")
+    input.input#email(type='text' v-model='$v.email.$model')
```
Comment (BJordi) [inline wolox-books/src/components/Login.vue]: The id goes before the class:
```js
input#email.input
```

File: wolox-books/src/components/Login.vue [hunk 2]
```diff
@@ -1,28 +1,28 @@
...
+    .error(v-show='$v.email.$error')
       | Email is required
-    label.label
+    label.label(for='password')
       | Password
-    input.input(type='password' v-model='$v.password.$model')
-    .error(v-if="$v.password.$error")
+    input.input#password(type='password' v-model='$v.password.$model')
```
Comment (BJordi) [inline wolox-books/src/components/Login.vue]: Add `id='password'`

File: wolox-books/src/components/Login.vue [hunk 3]
```diff
@@ -1,28 +1,28 @@
...
+    .error(v-show='$v.password.$error')
```
Comment (BJordi) [inline wolox-books/src/components/Login.vue]: Don't use a div to contain text.

File: wolox-books/src/components/Login.vue [hunk 4]
```diff
@@ -39,7 +39,10 @@ export default {
         'password': this.password
       }
       let response = await signIn(session);
-      console.log(response.data)
+      if (response.ok) {
+        setSessionData(response.data)
+        this.$router.push('/')
```
Comment (BJordi) [inline wolox-books/src/components/Login.vue]: Use the name of the route like this: `this.$router.push({ name: routes.home })`

File: wolox-books/src/components/NavBar.vue [hunk 5]
```diff
@@ -0,0 +1,42 @@
+<template lang='pug'>
+  .header.row
+    .img-box.column
+      img.wolox-img(src='@/assets/LogoWolox.png')
```
Comment (BJordi) [inline wolox-books/src/components/NavBar.vue]: Add an alternative text.

File: wolox-books/src/components/NavBar.vue [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+      span
```
Comment (BJordi) [inline wolox-books/src/components/NavBar.vue]: No class? :thinking:

File: wolox-books/src/components/NavBar.vue [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+        | BOOKS
+    a.logout(href='javascript:void(0)' @click='logout')
```
Comment (BJordi) [inline wolox-books/src/components/NavBar.vue]: What's that href? :thinking:

Comment (BJordi) [inline wolox-books/src/components/NavBar.vue]: Use the anchor (tag `a`) for links. If this is going to be something you click on, then, use a button with `type='button'`.

File: wolox-books/src/components/NavBar.vue [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+
+<script>
+import { removeSession } from '../services/userService'
+
+export default {
+  name: 'NavBar',
+  methods: {
+    logout () {
+      removeSession()
+      this.$router.push('/login')
```
Comment (BJordi) [inline wolox-books/src/components/NavBar.vue]: Use the name.

File: wolox-books/src/components/NavBar.vue [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+    }
+  }
+}
+</script>
+
+<style scoped lang='scss'>
+@import '../scss/variables/colors';
+.header {
```
Comment (BJordi) [inline wolox-books/src/components/NavBar.vue:29]: Add an empty line.

File: wolox-books/src/components/NavBar.vue [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+  methods: {
+    logout () {
+      removeSession()
+      this.$router.push('/login')
+    }
+  }
+}
+</script>
+
+<style scoped lang='scss'>
```
Comment (BJordi) [inline wolox-books/src/components/NavBar.vue]: Use double quotes in the html syntax.

File: wolox-books/src/components/NavBar.vue [hunk 11]
```diff
@@ -0,0 +1,42 @@
+<template lang='pug'>
```
Comment (BJordi) [inline wolox-books/src/components/NavBar.vue]: Use double quotes in the html syntax.

File: wolox-books/src/router.js [hunk 12]
```diff
@@ -9,7 +9,7 @@ export default new Router({
     {
       path: '/',
       name: routes.home,
```
Comment (BJordi) [inline wolox-books/src/router.js:11]: This shouldn't be a view, but remember to always match the name of the route with the view.

File: wolox-books/src/views/SignUp.vue [hunk 13]
```diff
@@ -1,38 +1,39 @@
...
+  form.column.container(@submit.prevent='submit')
     img.wolox-img(src='@/assets/LogoWolox.png')
     span.books-title
       | BOOKS
-    label.label
+    label.label(for='firstName')
       | First name
-    input.input(type='text' v-model='firstName')
-    label.label
+    input.input#firstName(type='text' v-model='firstName')
```
Comment (BJordi) [inline wolox-books/src/views/SignUp.vue]: id should go before classes:
```js
input#firstName.input
```

File: wolox-books/src/views/SignUp.vue [hunk 14]
```diff
@@ -1,38 +1,39 @@
...
+    label.label(for='lastName')
       | Last name
-    input.input(type='text' v-model='lastName')
-    label.label
+    input.input#lastName(type='text' v-model='lastName')
```
Comment (BJordi) [inline wolox-books/src/views/SignUp.vue]: Same here.

File: wolox-books/src/views/SignUp.vue [hunk 15]
```diff
@@ -1,38 +1,39 @@
...
+    label.label(for='email')
       | Email
-    input.input(type='text' v-model='email')
-    .error(v-show='emailRequired')
+    input.input#email(type='text' v-model='$v.email.$model')
```
Comment (BJordi) [inline wolox-books/src/views/SignUp.vue]: And here.

File: wolox-books/src/views/SignUp.vue [hunk 16]
```diff
@@ -1,38 +1,39 @@
...
       | Email is required
-    .error(v-show='invalidEmail')
+    span.error(v-show='invalidEmail')
       | Email is invalid
-    label.label
+    label.label(for='password')
       | Password
-    input.input(type='password' v-model='password')
-    .error(v-show='passwordRequired')
+    input.input#password(type='password' v-model='$v.password.$model')
```
Comment (BJordi) [inline wolox-books/src/views/SignUp.vue]: And here.

File: wolox-books/src/views/SignUp.vue [hunk 17]
```diff
@@ -1,38 +1,39 @@
...
+    span.error(v-show='passwordRequired')
       | Password is required
-    .error(v-show='invalidPassword')
-    input.input.signup-button(v-on:click='submit' type='submit' value='Sign Up')
+    span.error(v-show='invalidPassword')
+      | Password is invalid
+    button.input.signup-button(type='submit')
```
Comment (BJordi) [inline wolox-books/src/views/SignUp.vue:22]: No click event? :thinking:

File: wolox-books/src/views/SignUp.vue [hunk 18]
```diff
@@ -1,38 +1,39 @@
...
+      | Sign Up
     .login-container
       router-link.input.login-button(to='login')
```
Comment (BJordi) [inline wolox-books/src/views/SignUp.vue]: Use name of routes.

**Review by BJordi — CHANGES_REQUESTED** [review]

File: wolox-books/src/views/SignUp.vue (hunk 17 above)
Comment (gzamudio) [reply wolox-books/src/views/SignUp.vue:22]: i used `form.column.container(@submit.prevent='submit')` and it's working... it's wrong?

Comment (BJordi) [reply wolox-books/src/views/SignUp.vue:22]: No, it's perfect!

**Review by BJordi — APPROVED** [review]

**Pull request closed without merging**